include/aws/http/http2_stream_manager.h

```diff
@@ -160,5 +161,16 @@ void aws_http2_stream_manager_acquire_stream(
     struct aws_http2_stream_manager *http2_stream_manager,
     const struct aws_http2_stream_manager_acquire_stream_options *acquire_stream_option);
 
+/**
+ * Fetch the current metric from stream manager.
+ *
+ * @param http2_stream_manager
+ * @param out_metric The metric to be fetched
+ */
+AWS_HTTP_API
+void aws_http2_stream_manager_fetch_metric(
+    struct aws_http2_stream_manager *http2_stream_manager,
```

Comment:
const

Reply:
trivial: is this non-const because you need to acquire the lock?
If so, I'd be OK with making this const, and then just casting away the const-ness when you need to futz with the locks.

In principal it's const